lib/writers/json-writer.js

```diff
@@ -0,0 +1,407 @@
+                                object: 'block'
+                            })
+                        }
+                        
+                    } else {
+                        pEl.nodes[listIndex].nodes.push({
+                            data: {},
+                            type: 'listItemChild',
+                            nodes: [textObj],
+                            object: 'block'
+                        })
+                    }
+                    
+
+                } else if(pEl && _.has(pEl.nodes[pElIndex], 'marks') && pEl.nodes[pElIndex].marks.length > 0) {
+                    pEl.nodes[pElIndex].text = value;
+                } else if(pEl && !isEmpty(get(pEl.nodes[pElIndex], 'data.href'))) {
+                    const pTextIndex = pEl.nodes[pElIndex].nodes.length === 0 ? 0 : pEl.nodes[pElIndex].nodes.length - 1;
+                    if(pEl.nodes[pElIndex].nodes[pTextIndex].text === '') {
+                        pEl.nodes[pElIndex].nodes[pTextIndex].text = value;
+                    } else {
+                        pEl.nodes[pElIndex].nodes.push(textObj);
+                    }
+                   
+                } else {
+                    pEl.nodes.push(textObj);
+                    if(!isEmpty(get(parentEl.nodes[trIndex].nodes[tdIndex], 'data'))) {
+                        const colspan = get(parentEl.nodes[trIndex].nodes[tdIndex], 'data.colspan');
+                        // const rowspan = get(parentEl.nodes[trIndex].nodes[tdIndex], 'data.rowspan');
+                        if(colspan > 1) {
+
+                            for(let i = 1; i <= colspan - 1; i++) {
+                                parentEl.nodes[trIndex].nodes.push({
+                                    type: 'table_cell',
+                                    data: {
+                                        isMerged: true,
+                                        mergeCentre: {
+                                            x: 1,
+                                            y: 0
+                                        },
+                                        mergeDirection: {
+                                            right: true
+                                        },
+                                    },
+                                    object: 'block',
+                                    nodes: [{
+                                        data: {},
+                                        type: 'paragraph',
+                                        nodes: [{
+                                            text: '',
+                                            marks: [],
+                                            object: 'text'
+                                        }],
+                                        object: 'block'
+                                    }],
+                                })
+                            }
+                        } 
+                        // if(rowspan > 1) {
```

Comment:
Do we need this?

Reply:
Yeah, this is in progress. Logic with rowspan is I need to add new table_row object after the current node, but issue is I need to find current active node with current children iteration after adding it (xN).